doc/development-guide.md

```diff
@@ -502,4 +588,60 @@ public class ServiceNameInterceptor implements Interceptor {
 ## Plugin Unit Test
 * [Plugin Unit Test](plugin-unit-test.md)
 
-
+## EaseAgent Plugin Debug FAQ
+The above basically covers all aspects of plugin development, the following are common debugging issues during plugin development, the following plugin debugging FAQ can also be found in the section starting at 1:05:57 in the [video](https://www.youtube.com/watch?v=u6Aoa2roGuA).
```

Comment:
Youtube video is the Chinese language, we can put that in reference. And let's write the details here.

Reply:
ok, the video link has been moved to reference, and an image has been add to illustrate debug breakpoints.